scripts/mysql_install_db.sh

```diff
@@ -409,8 +414,21 @@ then
 fi
 
 # Create database directories
-for dir in "$ldata" "$ldata/mysql" "$ldata/test"
+if test "$skip_install_test_db" -eq 0
+then
+  extra_db="$ldata/test"
+else
+  extra_db=""
+  install_params="$install_params
+SET @skip_install_test_db=$skip_install_test_db;"
+fi
+
+for dir in "$ldata" "$ldata/mysql" "$extra_db"
 do
+  if test -z "$dir"
+  then
+    continue
+  fi
```

Comment:
Any clue why do we need to mkdir database directories instead of just doing `CREATE DATABASE mysql`?

Reply:
I don't know. I did find this code odd too but haven't tested ```CREATE DATABASE mysql```

Reply:
Could you give it a try? Though you may consider this request unrelated and ignore it.

Reply:
That is we should create datadir, but creation of `mysql` and `test` should be moved to the sql script.

Reply:
I'm working on that now...

Reply:
that worked as per amended commit. Well spotted.